mscore/timeline.cpp

```diff
@@ -2247,18 +2328,55 @@ void Timeline::updateGrid()
       viewport()->update();
       }
 
+//---------------------------------------------------------
+//   updateGridFromCmdState
+//---------------------------------------------------------
+
+void Timeline::updateGridFromCmdState()
+      {
+      if (!isVisible())
+            return;
+
+      if (!_score) {
+            updateGridFull();
+            return;
+            }
+
+      const CmdState& state = _score->cmdState();
+
+      const bool layoutChanged = state.layoutRange();
+
+      if (!layoutChanged) {
+            updateGridView();
+            return;
+            }
+
+      const bool layoutAll = layoutChanged && (state.startTick().negative() || state.endTick().negative());
```

Comment:
See https://travis-ci.org/musescore/MuseScore/jobs/652005292#L2630-L2636, "`class Ms::Fraction` has no member named `negative`"

Reply:
Looks like I forgot to commit some changes, as always. Will update PR soon.

Reply:
Any update on this?